options/options.go

```diff
@@ -80,6 +81,9 @@ type TerragruntOptions struct {
 	// If you want stderr to go somewhere other than os.stderr
 	ErrWriter io.Writer
 
+	// Stores output of auto-init so it can be removed later form other streams
+	InitStream bytes.Buffer
```

Comment:
Perhaps there should be an `AutoInitWriter` and `AutoInitErrWriter` here instead? By default, these would be set to the same as `Writer` and `ErrWriter`. However, for `xxx-all` commands, these could be set to separate values that write to `stdout` / `stderr` but are not stored in those streams later used to print clean error messages?

Also, should this value be in the `Clone` method? If not, explicitly add a comment there explaining why.